recipes/cudatoolkit/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
```

Comment:
Can you add a `set -ex` lin here please, so that we can debug this? Thanks!

Reply:
sure! thanks!